.github/workflows/build-test.yml

```diff
@@ -36,7 +36,9 @@ jobs:
         run: dotnet build --configuration Release BTCPayApp.Server
       # Setup infrastructure
       - name: Start containers
-        run: docker compose -f "submodules/btcpayserver/BTCPayServer.Tests/docker-compose.yml" up -d dev
+        run: |
+          docker compose -f "submodules/btcpayserver/BTCPayServer.Tests/docker-compose.yml" build
+          docker compose -f "submodules/btcpayserver/BTCPayServer.Tests/docker-compose.yml" up -d dev
       - name: Start BTCPay
```

Comment:
Can you remove this it didn't solve the issue, tracking issue #229

Reply:
Okay sure.. It solves for the sshd issue 

"No such image: btcpayservertests-sshd:latest" by building first before running. 

I guess it would be added in the tracking issue, I would remove from here

Reply:
Okay yeah but that issue , from what I've seen only happens when a PR is merged. Let me merge it and see if it fixes it.